stylesheets/style.css

```diff
@@ -0,0 +1,125 @@
+body {
+    margin: 0;
+    font-family: 'Space Mono', monospace;
+    display: grid;
+    grid-template-columns: 20% auto 20%;
+    grid-template-rows: auto 1fr auto;
+    color: #3e3e3e;
+    font-weight: 400;
+    background-color: whitesmoke;
+    min-height: 100vh;
+}
+
+header {
+    margin-top: 4em;
+    margin-bottom: 1.5em;
+    grid-column: 2/3;
+    text-align: center; 
+}
```

Comment:
It looks like there are at least a couple CSS rules duplicated between the 2 stylesheets. Since we can link multiple stylesheets per page, I recommend creating one style sheet to contain all styles shared across the pages, and individual stylesheets for pages that need specific rules. This lets us remove duplicated CSS rules, making long term maintenance easier and removing places we could create bugs.